### PR TITLE
Reporting Observer update

### PR DESCRIPTION
**Pull request by ebidel**
Still being reviewed internally but posting the code here to get things moving.

**Target Live Date:** m69 beta blog post

- [ ] This has been reviewed and approved by (NAME)
- [ ] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

**Comment by ebidel** [comment]
Ready for review. Been reviewed by various folks in the gdoc.

**Comment by ebidel** [comment]
`Blink>ReportingObserver` is a new component name and has yet to show up on https://blinkcomponents-b48b5.firebaseapp.com/.

File: src/content/en/updates/2018/07/reportingobserver.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+The callback can be used to send reports to a backend or analytics provider
+for further analysis.
+
+Why is that useful? Until now, deprecation and
+intervention warnings were only available in the DevTools as console messages.
+Interventions in particular are only triggered by various real-world constraints
+like device and network conditions. Thus, you may never even see these messages
+when developing/testing a site locally. `ReportingObserver` provides
+the solution to this problem. When users experience potential issues in the wild,
+we can be notified about them.
```
Comment (samdutton) [inline src/content/en/updates/2018/07/reportingobserver.md]: Nit: maybe 'you can be notified'?

File: src/content/en/updates/2018/07/reportingobserver.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+    // → report.body.anticipatedRemoval === <JS_DATE_STR> or null
+  }
+}});
+
+observer.observe();
+```
+
+#### Filtered reports {: #filtered }
+
+Reports can be pre-filter to only observe certain report types:
```
Comment (samdutton) [inline src/content/en/updates/2018/07/reportingobserver.md]: 'pre-filtered'?

Also, are the reports filtered or the observation not done for types not included?

**Pull request merged**